example/storybook/src/components/Forms/FormControl/index.stories.mdx

```diff
@@ -275,6 +301,607 @@ The following are the lists of all the libraries and packages the component reli
+      scope: {
+        Wrapper,
+        FormControl,
+        Icon,
+        Radio,
+        WarningIcon,
+        VStack,
+        CircleIcon,
+        Heading,
+      },
+      argsType: {},
+    }}
+  />
+</AppProvider>
+
+#### Form Control with Checkbox
+
+<AppProvider>
+  <CodePreview
+    showComponentRenderer={true}
+    showArgsController={false}
+    metaData={{
+      code: `
+        <FormControl>
+          <FormControl.Label>
+            <Heading color='$text900' fontSize='$md' lineHeight='$md'>
+              Sign up for newsletters
+            </Heading>
+          </FormControl.Label>
+          <Checkbox.Group>
+            <VStack space="sm">
+              <Checkbox size="sm" value="Mango">
+                <Checkbox.Indicator>
+                  <Checkbox.Icon>
+                    <CheckIcon />
+                  </Checkbox.Icon>
+                </Checkbox.Indicator>
+                <Checkbox.Label fontSize='$sm' lineHeight='$md' color='$text900'>
+                  Daily Bits
+                </Checkbox.Label>
+              </Checkbox>
+              <Checkbox size="sm" value="Apple">
+                <Checkbox.Indicator>
+                  <Checkbox.Icon>
+                    <CheckIcon />
+                  </Checkbox.Icon>
+                </Checkbox.Indicator>
+                <Checkbox.Label fontSize='$sm' lineHeight='$md' color='$text900'>
+                  Event Updates
+                </Checkbox.Label>
+              </Checkbox>
+              <Checkbox size="sm" value="Orange">
+                <Checkbox.Indicator>
+                  <Checkbox.Icon>
+                    <CheckIcon />
+                  </Checkbox.Icon>
+                </Checkbox.Indicator>
+                <Checkbox.Label fontSize='$sm' lineHeight='$md' color='$text900'>
+                  Sponsorship
+                </Checkbox.Label>
+              </Checkbox>
+            </VStack>
+          </Checkbox.Group>
+          <FormControl.Helper mt='$2'>
+            <FormControl.Helper.Text fontSize='$sm' lineHeight='$md'>
+              Subscribe to newsletters for updates.
+            </FormControl.Helper.Text>
+          </FormControl.Helper>
+        </FormControl>
+      `,
+      transformCode: (code) => {
+        return transformedCode(code);
+      },
+      scope: {
+        Wrapper,
+        FormControl,
+        Icon,
+        Checkbox,
+        WarningIcon,
+        VStack,
+        CheckIcon,
+        Heading,
+      },
+      argsType: {},
+    }}
+  />
+</AppProvider>
+
+#### Form Control with TextArea
+
+<AppProvider>
+  <CodePreview
+    showComponentRenderer={true}
+    showArgsController={false}
+    metaData={{
+      code: `
+        <FormControl>
+          <FormControl.Label mb='$1'>
+            <FormControl.Label.Text color='$text500' lineHeight='$md'>
+              Comment
+            </FormControl.Label.Text>
+          </FormControl.Label>
+          <TextArea>
+            <TextArea.Input pt='$2' pr='$1'/>
```

Comment:
Reason for right padding?

Reply:
to get the padding exact as the design.
in figma, there is different padding for left and right side inside the textArea.

Reply:
design has been corrected.

Reply:
updated